p2p/p2p.go

```diff
@@ -139,98 +142,70 @@ func (s *Service) sendStatusMessages(peer peer.ID) {
 	}
 }
 
-// broadcastReceivedMessages starts polling the msgRec channel for messages
-// from the core service and broadcasts the new messages to connected peers
-func (s *Service) broadcastReceivedMessages() {
+// broadcastMessages broadcasts messages from the core service
+func (s *Service) broadcastMessages() {
```

Comment:
not a great name, I think "broadcast" should be dropped from the name, maybe "receiveCoreMessages"?

Reply:
True. Not a great name. Haha. I like `receiveCoreMessages`. :+1:

Reply:
fixed